feature/manager.go

```diff
@@ -95,3 +95,14 @@ func (m *Manager) Get(set Set) *lnwire.FeatureVector {
 	raw := m.GetRaw(set)
 	return lnwire.NewFeatureVector(raw, lnwire.Features)
 }
+
+// List returns a list of the feature vectors for our node.
+func (m *Manager) List() []*lnwire.FeatureVector {
+	var vectors []*lnwire.FeatureVector
+
+	for set := range m.fsets {
+		vectors = append(vectors, m.Get(set))
```

Comment:
another option here is to add `feature.SetList`, and then add this in each feature's set descriptor. might simplify the rpc conversion for self node